.github/workflows/ci.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       - name: cargo install cargo-llvm-cov
         uses: taiki-e/install-action@cargo-llvm-cov
       - name: cargo llvm-cov
-        run: cargo llvm-cov --all-features --lcov --output-path lcov.info
+        run: cargo llvm-cov --all-features --lcov --output-path lcov.info --no-cfg-coverage --no-cfg-coverage-nightly
```

Comment:
these options are needed because `rand` now uses `zerocopy` which internally uses the unstable [`cfg(coverage)`](https://github.com/rust-lang/rust/issues/84605). once this is stabilized, then I believe these options can be removed